components/Heading/index.jsx

```diff
@@ -32,27 +32,25 @@ function Heading({ tag, showAnchorIcons, ...props }) {
   return React.createElement(tag, attrs, children);
 }
 
-function CreateHeading(level, anchors, { showAnchorIcons }) {
+function CreateHeading(level, { showAnchorIcons }) {
   // eslint-disable-next-line react/display-name
-  return props => (
-    <Heading {...props} anchors={anchors} level={level} showAnchorIcons={showAnchorIcons} tag={`h${level}`} />
-  );
+  return props => <Heading {...props} level={level} showAnchorIcons={showAnchorIcons} tag={`h${level}`} />;
```

Comment:
This change is entirely unrelated, but AFAICT `level` was not being used anymore???